File: poo/3. Newton's Laws Simulator/force.php
<?php
    class force extends forces {
        readonly private int $mass;
        readonly private int $acceleration;
        private int $force;

        public function __construct(int $mass, int $acceleration) {
            parent::__construct("Fuerza", "Fuerza");
            $this->mass = $mass;
            $this->acceleration = $acceleration;
        }
        public function getMass(): int {
            return $this->mass;
        }
        public function getAcceleration(): int {
            return $this->acceleration;
        }

        public function calculateForce(): ?float{
            $this->force = $this->mass * $this->acceleration;
            return $this->force;
        }
    }
?>

File: poo/3. Newton's Laws Simulator  JS/force.java
public class Force extends Forces {
    private final int mass;
    private final int acceleration;
    private int force;

    public Force(int mass, int acceleration) {
        super("Fuerza", "Fuerza");
        this.mass = mass;
        this.acceleration = acceleration;
    }

    public int getMass() {
        return mass;
    }

    public int getAcceleration() {
        return acceleration;
    }

    public Float calculateForce() {
        this.force = this.mass * this.acceleration;
        return (float) force;
    }
}
